public_html/wp-content/mu-plugins/load-other-mu-plugins.php

```diff
@@ -24,6 +24,10 @@ function wcorg_include_individual_mu_plugins() {
 		require_once( __DIR__ . '/blocks/blocks.php' );
 	}
 
+	if ( is_readable( __DIR__ . '/vendor/autoload.php' ) ) {
```

Comment:
`MIGHT` - Are you checking `is_readable()` so that local installs won't fatal before running `composer install`?

it seems like it might be good to let it fatal, so that they can see something is wrong, and hopefully check the docs to see that they need to run it.

Or would the fatal interfere with some part of the automated provision process?

Reply:
🤔 It shouldn't interfere with anything. But the fatal error message might not be informative enough for someone to understand that they need to run `composer install`.

We could add an `else` here and then throw a custom fatal error message. Something like:

```php
wp_die( 'Required files are missing. Please run <code>composer install</code> and try again.' );
```

Reply:
That's fair.

The main benefit of a change, from my perspective , was just reducing the lines of code, so the current approach and adding `wp_die()` feel about the same to me. If you have a preference, that WFM.

Reply:
Done in 99517c82